tests/integration/security/testdata/authz/egress-gateway.yaml.tmpl

```diff
@@ -13,12 +13,12 @@ spec:
 apiVersion: security.istio.io/v1
 kind: AuthorizationPolicy
 metadata:
-  name: egressgateway
+  name: {{.EgressGatewayServiceName | default "egressgateway"}}
```

Comment:
nit: this technically doesn't need this. But doesn't really matter, the name can be anything

Reply:
Hey John, can you expand on why we may not need it?

Reply:
The name doesn't matter, it could be `foo`. I don't mind the change either, but should be fine without